### ISSUE TITLE
Let hyperion's output be human readable

### ISSUE THREAD

**Issue by aspiwack** [tweag/hyperion#9]
This is linked to #6 somehow.

The issue is that, currently, hyperion outputs raw json on the standard output. This is not particularly readable, despite the results being immediately useful. We should be careful of not doing the same mistake as other tools and giving the choice between the human-readable output and the computer-readable output: I want to be able to quickly read the result of the current benchmark as well as produce json file for ingestion in a database. So we should be able to specify an output file for the json output and print a nice summary on the stdout.

I'm agnostic regarding the way the output is presented. Core bench gives a table of the form
```
+-------+-----------+-----------+--
|  Name | Analysis1 | Analysis2 | …
+-------+-----------+-----------+--
| name1 | a11       | a21       |
| name2 | a12       | a22       |
```
An alternative presentation which is just as good would be (including formatting and indentation):

**name1:**
  Analysis1: a11
  Analysis2: a21
**name2:**
  Analysis1: a12
  Analysis2: a22

**Comment by qnikst** [comment]
I totally agree, and I think it's very close to what #6 is asking but put in better form. 
What do you think about UI here, though?
As far as I understand you want the full machine output to be always be written, so seems that we need to control where it was written, i.e. have an option to set filename, and have a reasonable default if option was not passed. Do you have an idea for that reasonable default, and possibly other problems that can be here?

**Comment by gamazeps** [comment]
@qnikst @aspiwack a simple approach would be to separate the two outputs.

Adding the `--pretty` flag would output the human readable format to `stdout`, all the while using `--output` and the `--raw` options mentionned in #6 would do their things.

Does that sound good ? i.e. if --pretty is passed, give nice output on stdout (and keep ugly output if no file is provided for it)

**Comment by gamazeps** [comment]
Closed by #10
